google/cloud/forseti/services/inventory/base/gcp.py

```diff
@@ -41,11 +41,11 @@ class ApiClient(object):
     __metaclass__ = abc.ABCMeta
 
     @abc.abstractmethod
-    def fetch_bigquery_dataset_policy(self, project_id, dataset_id):
+    def fetch_bigquery_dataset_policy(self, project_number, dataset_id):
         """Dataset policy Iterator for a dataset from gcp API call.
 
         Args:
-            project_id (str): id of the project to query.
+            project_number (str): id of the project to query.
```

Comment:
If this is `project_number` now, can we also update the `id` the description?

Reply:
Right now saying 'id' is consistent across all docstrings, do you think this should be globally changed in this PR, or a future one?